.env.example

```
@@ -0,0 +1,10 @@
+# 해당 파일을 복사해서 .env 를 만들고 실제 값을 채워주세요
+# .env 는 커밋되지 않습니다 (.gitignore)
+
+VITE_API_BASE_URL=# 백엔드 API 주소
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)